src/database/schemamanager.cpp

```diff
@@ -129,14 +114,11 @@ SchemaManager::Result SchemaManager::upgradeToSchemaVersion(
         revisionMap[iVersion] = revision;
     }
 
-    // The checks above guarantee that currentVersion < targetVersion when we
-    // get here.
     while (m_currentVersion < targetVersion) {
         int nextVersion = m_currentVersion + 1;
 
-        // Now that we bake the schema.xml into the binary it is a programming
-        // error if we include a schema.xml that does not have information on
-        // how to get all the way to targetVersion.
+        // With the packaged schema.xml it is an error if it does not have
+        // information on how to get all the way to targetVersion.
```

Comment:
I think the previous comment was more clear.

Reply:
I wanted to omit the information that we "now" include it - the past state does not matter anymore. Now that I think about it, maybe we delete this comment entirely? The error messsage should be self-explanatory.

Reply:
ok